__tests__/integration.test.js

```diff
@@ -181,47 +181,107 @@ describe("POST: /api/articles/:article_id/comments", () => {
       });
   });
   test("POST: responds with a 201 status code, with the correct object structure", () => {
-      const newComment = {
-        username: "butter_bridge",
-        body: "This article is amazing, 100% recommend",
-      };
-      return request(app)
+    const newComment = {
+      username: "butter_bridge",
+      body: "This article is amazing, 100% recommend",
+    };
+    return request(app)
       .post("/api/articles/1/comments")
       .send(newComment)
-      .then(({body}) => {
-        const { newComment } = body
+      .then(({ body }) => {
+        const { newComment } = body;
         expect(newComment.comment_id).toBe(19);
         expect(newComment.body).toBe("This article is amazing, 100% recommend");
         expect(newComment.article_id).toBe(1);
         expect(newComment.author).toBe("butter_bridge");
         expect(newComment.votes).toBe(0);
         expect(typeof newComment.created_at).toBe("string");
-      })
-  })
+      });
+  });
   test("POST: responds with a 400 status code when provided an ID which does not exist in the database", () => {
     return request(app)
-    .post("/api/articles/11111/comments")
-    .send({
-      username: "Do_not_exist",
-      body: "hello" 
-    })
-    .expect(400)
-    .then((response) => {
-      const {body} = response
-      expect(body).toEqual({ message: "bad request" });
-    })
-  })
+      .post("/api/articles/11111/comments")
+      .send({
+        username: "Do_not_exist",
+        body: "hello",
+      })
+      .expect(400)
+      .then((response) => {
+        const { body } = response;
+        expect(body).toEqual({ message: "bad request" });
+      });
+  });
   test("POST: responds with a 404 status code no ID is provided", () => {
     return request(app)
-    .post("/api/articles//comments")
-    .send({
-      username: "Do_not_exist",
-      body: "hello"
-    })
-    .expect(404)
-    .then((response) => {
-      const {body} = response
-      expect(body).toEqual({message: "Path does not exist"})
-    })
-  })
+      .post("/api/articles//comments")
+      .send({
+        username: "Do_not_exist",
+        body: "hello",
+      })
+      .expect(404)
+      .then((response) => {
+        const { body } = response;
+        expect(body).toEqual({ message: "Path does not exist" });
+      });
+  });
+});
+describe("PATCH: /api/articles/:article_id", () => {
+  test("PATCH: responds with a 200 status code when updated votes by increment of 1", () => {
+    let newVote = 1;
+    const articleVotes = { inc_votes: newVote };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(articleVotes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articleVotes.votes).toBe(101);
+      });
```

Comment:
Great that you have checked the votes property, but the task does require the whole article to be returned, so would be more thorough to check that the whole article object is returned, i.e. check more than just the votes key is returned.
For that reason it would be better to name the key the response is sent on something a bit clearer than `articleVotes`